src/test/java/uk/gov/hmcts/ccd/domain/service/lau/AuditCaseRemoteOperationTest.java

```diff
@@ -139,7 +139,7 @@ void shouldPostCaseSearchRemoteAuditRequest() {
         verify(httpClient).sendAsync(captor.capture(),any());
 
         assertThat(captor.getValue().uri().getPath(), is(equalTo("/caseSearch")));
-        assertThat(captor.getValue().headers().map().size(), is(equalTo(2)));
+        assertThat(captor.getValue().headers().map().size(), is(equalTo(3)));
         assertThat(captor.getValue().headers().map().get("ServiceAuthorization").get(0), is(equalTo("Bearer 1234")));
         assertThat(captor.getValue().headers().map().get("Content-Type").get(0), is(equalTo("application/json")));
```

Comment:
Can we assert the third value as well?

Reply:
I've provided the asserr for the Accept header, thank.